resources/zib/zib-Pregnancy-EstimatedDateOfDelivery.xml

```diff
@@ -0,0 +1,102 @@
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-pregnancy-v4.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/Pregnancy-v4.0(2020EN)" />
+    <name value="zib Pregnancy-v4.0(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation.effective[x]">
+      <path value="Observation.effective[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.effective[x]:effectiveDateTime">
+      <path value="Observation.effective[x]" />
+      <sliceName value="effectiveDateTime" />
+      <short value="DateOfEstimation" />
+      <definition value="Date on which the delivery date is estimated." />
+      <alias value="DatumBepaling" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="zib-pregnancy-v4.0-2020EN" />
+        <map value="NL-CM:7.14.11" />
+        <comment value="DateOfEstimation" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueDateTime">
+      <path value="Observation.value[x]" />
+      <sliceName value="valueDateTime" />
+      <short value="EstimatedDateOfDelivery" />
+      <definition value="The date on which the pregnancy is expected to be 40w 0d (280 days). Different due dates are used at various points in the pregnancy." />
+      <alias value="ATermeDatum" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="zib-pregnancy-v4.0-2020EN" />
+        <map value="NL-CM:7.14.3" />
+        <comment value="EstimatedDateOfDelivery" />
+      </mapping>
+    </element>
+    <element id="Observation.method">
+      <path value="Observation.method" />
+      <short value="EstimatingMethod" />
+      <definition value="Method by which the delivery date is estimated." />
+      <alias value="BepalingsMethode" />
+      <binding>
```

Comment:
Moet de omschrijving hier nog worden aangepast? Er staat nu "methods for simple observations" dat lijkt me te generiek?

Reply:
Er staat "Method by which the delivery date is estimated." dit is ook de definitie van EstimatingMethod?